trieste/acquisition/function/active_learning.py

```diff
@@ -245,3 +246,78 @@ def acquisition(x: TensorType) -> TensorType:
         return criterion
 
     return acquisition
+
+class BayesianActiveLearningByDisagreement(SingleModelAcquisitionBuilder):
+    """
+    Builder for the *Bayesian Active Learning By Disagreement* acquisition function defined in
+    :cite:`houlsby2011bayesian`. The acquisition function computes the information gain
+    of the predictive entropy.  
+
+    This acquisition function is intended to use for Binary Gaussian Process Classification
+    model with Bernoulli likelihood. integrating over nuisance parameters is currently not 
+    supported (see equation 6 of the paper)
+    """
+
+    def __init__(self, jitter: float = DEFAULTS.JITTER) -> None:
+        """
+        :param jitter: The size of the jitter to avoid numerical problem caused by the
+                log operation if variance is close to zero.
+        """
+        self._jitter = jitter
```

Comment:
this should be > 0, right? then lets assert its scalar and positive and have a test for raising the exceptions

Reply:
added, test will be added later along with another tests